nfts/frontend/src/App.js

```diff
@@ -1,10 +1,31 @@
-import logo from "./logo.svg";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Navbar from "./Components/Navbar/Navbar";
+import { MetaKeep } from "metakeep";
+import { TransferNFT } from "./Components/NFT/TransferNFT";
 import "./App.css";
+import { useMemo, useState } from "react";
 
 function App() {
+  const [env, setEnv] = useState("dev");
+  const [apiKey, setApiKey] = useState("");
```

Comment:
this is appId

Reply:
You can actually remove app id. It's not needed for the consent flow.

Reply:
fixed